controllers/designate_controller.go

```diff
@@ -762,25 +761,14 @@ func (r *DesignateReconciler) reconcileNormal(ctx context.Context, instance *des
 		allocatedIPs[predIP] = true
 	}
 
-	// Get a list of the nodes in the cluster
-
-	// TODO(oschwart):
-	// * confirm whether or not this lists only the nodes we want (i.e. ones
-	// that will host the daemonset)
-	// * do we want to provide a mechanism to temporarily disabling this list
-	// for maintenance windows where nodes might be "coming and going"
-
-	nodes, err := helper.GetKClient().CoreV1().Nodes().List(ctx, metav1.ListOptions{})
-	if err != nil {
-		return ctrl.Result{}, err
-	}
-
-	var nodeNames []string
-	for _, node := range nodes.Items {
-		nodeNames = append(nodeNames, fmt.Sprintf("mdns_%s", node.Name))
+	// Handle Mdns predictable IPs configmap
+	mdnsReplicaCount := int(instance.Spec.DesignateMdns.Replicas)
```

Comment:
should be

`mdnsReplicaCount := int(i*nstance.Spec.DesignateMdns.Replicas)`

Reply:
Thanks, I have just fixed it.